hackathon2.R

```diff
@@ -0,0 +1,90 @@
+# GOAL of this research: ARE WE IMPROVING on SCREENING of MUAC and OEDEMA? 
+##Are wee expanding our screening coverage? From baseline vs. endline? By sex? By location?
+
+## Install & Load R packages 
+install.packages("gtsummary")
+install.packages("dplyr")
+
+library(dplyr)
+library("gtsummary")
+
+## Load functions in R directory 
+for (f in list.files("R", full.names = TRUE)) source (f)
+
+# Read data 
+nut<-read.csv(file="https://raw.githubusercontent.com/OxfordIHTM/ihtm-hackathon-2023/main/data/nutrition_survey_data.csv")  
+nut
+
+##Subset data: Transform variables of interest into table nut1 
+nut1 <- nut %>% select(survey_round,county,sex,muac_screen,oedema_screen)
```

Comment:
Again, this is syntactically correct. But from an analysis perspective, this step isn't really needed.

A good mantra for data analysis is to not take away structure and shape of original dataset as much as possible. Selecting only the variables you needed isn't necessarily wrong but for the rest of your script, it isn't also needed. You can still do the transformations you want to do without having to select out the other variables. Those other variables may not be of interest to you in terms of the questions you are trying to answer but it can be of interest later on if you want to expand your analysis. For example, what if you later on decide to analyse the probability of a SAM child in the sample to have been screened in the past month and also being in the programme to show whether MUAC and oedema screening increases the chance of a SAM child being covered by the programme? When you go back to your code, you will have to bring back those other variables rather than just continuing on with your work with the variables already being there.

The only time that we de-select a variable is if we need to anonymise data or there are restricted variables that we need to remove from the data. Or if we are at a stage where we need to create pretty reporting tables for specific variables. Otherwise, it makes more sense to keep them all in.